rdkit/Chem/inchi.py

```diff
@@ -0,0 +1,161 @@
+#
```

Comment:
This should not be in source control.
If you want to do a cleanup of `inchi.py`, the place to look is `External/INCHI-API/python/inchi.py`